<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddPadDialog</class>
 <widget class="QDialog" name="CAddPadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="round_radio_btn">
       <property name="text">
        <string>Round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rect_radio_btn">
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="width_lbl">
         <property name="text">
          <string>Width : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="width_line_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="height_lbl">
         <property name="text">
          <string>Height :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="height_line_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="setting_btn">
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
